src/conventions.jl

```diff
@@ -0,0 +1,33 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+Convention used to reconstruct the faces of a polytope
+from a list of vertices.
+"""
+abstract type OrderingConvention end
+
+"""
+    connectivity(polytope, convention, k)
```

Comment:
Perhaps the name of this function should be `connectivities` instead? You are returning a list, which is a plural, not singular concept.

Reply:
I was thinking that connectivity would express the set of connections, how things connect.

Reply:
When you restructure the PR tomorrow by placing the functions in places near the concepts they belong, we will be able to find an appropriate name for it :ok_hand: